### ISSUE TITLE
F-droid

### ISSUE THREAD

**Issue by flashtheman** [breez/breezmobile#413]
Is there some way to install breez without having google play on Android? I didn't find the app on F-droid.

**Comment by tifDev** [comment]
This would be nice

**Comment by AbelLykens** [comment]
Would be good. Any other way of getting the APK would be useful too.

**Comment by emanuelb** [comment]
Please open RFP (Request For Packaging) for this app in F-Droid repo: https://gitlab.com/fdroid/rfp#submission-queue which is the first step for f-droid inclusion.

**Comment by n4HeVQSGqDeEu6** [comment]
This is the main reason I'm not using Breez at the moment. Many bitcoiners use an alternative OS on their phones (like grapheneos)...

**Comment by kingonly** [comment]
We support it via our APK: https://doc.breez.technology/Can-I-download-Breez-APK.html

**Comment by AbelLykens** [comment]
Do you expect users to manually check that page weekly? Or is there a built-in update check (there is no auto update, I know, but notifying about a new release at least would really help)?

**Comment by kingonly** [comment]
There's an in-app 'update' CTA

**Comment by daum3ns** [comment]
> There's an in-app 'update' CTA

yeah, but the link goes to the google playstore of the breeze app... this should go to the .apk download for ungoogled phones..

**Comment by kingonly** [comment]
> > There's an in-app 'update' CTA
> 
> yeah, but the link goes to the google playstore of the breeze app... this should go to the .apk download for ungoogled phones..

True, @ademar111190 please follow up. We should have the logic to identify it (@yaslama can help).
